Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -6948,5 +6948,172 @@ struct PackageCommandTests {
                 expectNoDiagnostics(observability.diagnostics)
             }
         }
+
+        @Test(arguments: getBuildData(for: SupportedBuildSystemOnAllPlatforms))
```

Comment:
It's ok to wrap this in something like this for now to get to a passing CI run:

```
withKnownIssue {
} when: {
                ProcessInfo.hostOperatingSystem == .windows
                    || (ProcessInfo.hostOperatingSystem == .linux && data.buildSystem == .swiftbuild)
            }
```

There are some unrelated issues setting up the runtime library paths on linux for plugin executables which I haven't fixed yet.

Reply:
@owenv can you rerun the CI again, please?